### PR TITLE
Added `datadog.ignoreAutoConfig` config option

### PR DESCRIPTION
**Pull request by yafernandes**
Recent addition of kubeStateMetricsCore can set DD_IGNORE_AUTOCONF directly, creating possible conflict with users setting the env var explicitly.  This PR goal is to make possible both to requirements to coexist.

### CODE REVIEW

File: charts/datadog/values.yaml
Comment (ahmed-mez) [inline charts/datadog/values.yaml:425]: we can provide an example here
```suggestion
  ignoreAutoConfig: []
  #  - redisdb
  #  - kubernetes_state
```

File: charts/datadog/values.yaml
Comment (ahmed-mez) [inline charts/datadog/values.yaml:424]:
```suggestion
  # datadog.ignoreAutoConfig -- List of integration to ignore auto_conf.yaml.
  ## ref: https://docs.datadoghq.com/agent/faq/auto_conf/
```

**Review by ahmed-mez — COMMENTED** [review]
Thank you @yafernandes  🙇 
two small comments!

**Comment by yafernandes** [comment]
Thank you @ahmed-mez .

**Review by ahmed-mez — COMMENTED** [review]
@yafernandes LGTM, we can merge it once you solve the merge conflict!

**Review by ahmed-mez — APPROVED** [review]

**Pull request merged**
